### PR TITLE
fix: introduce menu built-in keyboard support

### PR DESCRIPTION
**Pull request by JKMarkowski**
fixes: https://github.com/SAP/fundamental-ngx/issues/1062

### Please provide a brief summary of this pull request.
I added menu built-in keyboard support to combobox. And changed a little bit split button examples, cause there was visual bug with anchors color.
I also modified the tests, deleted some useless code.

#### If this is a new feature, have you updated the documentation?
I updated documentation for split button.

### CODE REVIEW

**Comment by MattL75** [comment]
Keyboard support is broken on last combobox example

File: library/src/lib/combobox/combobox.component.html [hunk 1]
```diff
@@ -29,15 +29,25 @@
...
+                           [innerHTML]="term | displayFnPipe:displayFn | highlight:inputText:highlighting">
+                        </a>
+                    </li>
+                </ng-container>
+                <ng-container *ngIf="itemTemplate">
+                    <li *ngFor="let term of displayedValues; let index = index;"
+                        (click)="onMenuClickHandler(index)"
+                        (keydown)="onMenuKeydownHandler($event, index)"
+                        fd-menu-item tabindex="0"
+                    >
```
Comment (MattL75) [inline library/src/lib/combobox/combobox.component.html]: Why did you change this? Why does there need to be two different loops in two different ng-containers?

Comment (MattL75) [reply library/src/lib/combobox/combobox.component.html]: Can just leave the fd-menu-item on the li? This change makes the file pretty bloaty and confusing

File: library/src/lib/combobox/combobox.component.ts [hunk 2]
```diff
@@ -145,14 +155,27 @@ export class ComboboxComponent implements ControlValueAccessor, OnInit, OnChange
         }
     }
 
+    ngOnDestroy(): void {
+        this.onDestroy$.next();
```
Comment (MattL75) [inline library/src/lib/combobox/combobox.component.ts]: complete the observable after .next

File: library/src/lib/combobox/combobox.component.html (hunk 1 above)
Comment (JKMarkowski) [reply library/src/lib/combobox/combobox.component.html]: We can, but then the anchors won't have fd-menu-item class. So it will have blue color.

Comment (JKMarkowski) [reply library/src/lib/combobox/combobox.component.html]: I will add @input to class host binding, to have a control on it, then add just a class to anchor.

Comment (MattL75) [reply library/src/lib/combobox/combobox.component.html]: Do we even need the anchors? I'm not sure why we have them, doesn't seem like a11y would require it

Comment (MattL75) [reply library/src/lib/combobox/combobox.component.html]: HostBinding is a decent solution in the mean time

Comment (JKMarkowski) [reply library/src/lib/combobox/combobox.component.html]: I just changed anchors to spans in combobox.

**Comment by JKMarkowski** [comment]
All comments are solved.

File: library/src/lib/menu/menu-item.directive.ts [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import { Directive, ElementRef, HostBinding } from '@angular/core';
+import { Directive, ElementRef, HostBinding, Input } from '@angular/core';
```
Comment (JKMarkowski) [inline library/src/lib/menu/menu-item.directive.ts]: Unused Input - to remove
